### PR TITLE
convert icons from a font to svg [PTO-5]

### PR DESCRIPTION
**Pull request by leejordan**
- remove font awesome
- set up svg icons only for icons that we actually use
- change link focus style to account for fact that icons are no longer a font and do not change with `color`

screenshots showing svg icons and new focus effect for links

screen shot 2016-03-09 at 13 02 40
screen shot 2016-03-09 at 13 02 49
screen shot 2016-03-09 at 13 03 07

### CODE REVIEW

File: less/common/icons.less [hunk 1]
```diff
@@ -2,18 +2,26 @@
...
-@import '@{font-awesome-path}/stacked';
-@import '@{font-awesome-path}/icons';
+// set up base icon class
+.fa {
+    .icon();
+    margin-top: -0.2em;
+}
+
+// icon modifiers
+.fa-2x {
```
Comment (notlee) [inline less/common/icons.less]: BEM it up?

File: less/includes/mixins.less [hunk 2]
```diff
@@ -401,20 +402,47 @@
...
-    font-variant: normal;
-    text-transform: none;
-    line-height: 1em;
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
+    line-height: 1;
+    background-size: contain;
+    background-position: center;
+    background-repeat: no-repeat;
+    speak: none;
```
Comment (notlee) [inline less/includes/mixins.less]: Speak. Interesting.

File: less/common/icons.less (hunk 1 above)
Comment (iamstuartwilson) [inline less/common/icons.less]: Better to remain consisten with font awesome here. Although, don't take this comment to serious Lee................. :tongue:

Comment (notlee) [inline less/common/icons.less]: Stuart's alive! :dancer: :smile: 

I disagree. Losing the parity with fa isn’t an issue, we aren’t likely to be going back and should hopefully have our own icons soon.

Comment (leejordan) [inline less/common/icons.less]: I could go either way on this but as we're planning to move to our own icons I suppose now would be a good time to break parity with fa.

**Pull request closed without merging**